### PR TITLE
Add tests for server

### PR DESCRIPTION
**Pull request by sofignatova**

### CODE REVIEW

**Comment by sofignatova** [comment]
@googlebot I signed it!

File: server-example/test_app.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import pytest
```
Comment (brianquinlan) [inline server-example/test_app.py:1]: Could you add a license to the top of this file? You can use any other file in the project as an example but make sure that you change the year to 2020.

Comment (brianquinlan) [reply server-example/test_app.py:1]: Also, every file needs a comment explaining what it is doing.

File: server-example/test_app.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+@pytest.fixture
+def client():
+    with app.app.test_client() as client:
+        yield client
+
+def test_image(client):
+    rv = client.get("/")
+    assert b'build' in rv.data
+    assert b'passing' in rv.data
```
Comment (brianquinlan) [inline server-example/test_app.py:13]: Add a newline at the end here.

File: server-example/requirements.txt [hunk 3]
```diff
@@ -1,2 +1,3 @@
 Flask>1.1
 pybadges
+pytest
```
Comment (brianquinlan) [inline server-example/requirements.txt:3]: Hmmmm....maybe it would be better to have a separate requirements file for tests? Like:
requirements-test.txt
-r requirements.txt
pytest

Also, could you hook this up to the nox tests? You probably have to edit `_run_tests` and `unit` in noxfile.py

File: noxfile.py [hunk 4]
```diff
@@ -33,24 +34,27 @@ def lint(session):
...
 @nox.session
 def unit(session):
     """Run the unit test suite."""
     session.install('-e', '.[dev]')
+    session.install('-r', 'server-example/requirements-test.txt')
     _run_tests(session)
 
 
-@nox.session(python=['3.4', '3.5', '3.6', '3.7'])
+@nox.session(python=['3.4', '3.5', '3.6', '3.7', '3.8'])
```
Comment (brianquinlan) [inline noxfile.py:50]: That should probably be in another PR but it's fine this time.

File: noxfile.py [hunk 5]
```diff
@@ -33,24 +34,27 @@ def lint(session):
     serious code quality issues.
     """
     session.install('flake8')
-    session.run('flake8',
-                'pypadges,tests')
+    session.run('flake8', 'pybadges')
```
Comment (brianquinlan) [inline noxfile.py:37]: That should probably be in another PR but it's fine this time.
